pkg/internal/build/gen_mk.sh

```
@@ -0,0 +1,228 @@
+PROJECTS="$(cut -d':' -f1 "$INDEX")"
+COMMANDS="tidy get build test up"
+
+TAB=$(printf "\t")
+
+escape_dir() {
+	echo "$1" | sed -e 's|/|\\/|g' -e 's|\.|\\.|g'
+}
+
+expand() {
+	local prefix="$1" suffix="$2"
+	local x= out=
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)